pytket/docs/conf.py

```diff
@@ -40,7 +40,7 @@
 # The short X.Y version
 version = "1.7"
 # The full version, including alpha/beta/rc tags
-release = "1.7.2"
+release = "1.7.3"
```

Comment:
Yeah weird, I'm also not sure. Maybe it's because I haven't pulled in from develop for a while - have done so now incase it nudges github to not show the diff.